.github/workflows/main.yml

```
@@ -0,0 +1,26 @@
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: AI Code Reviewer
+        uses: villesau/ai-codereviewer@v2.7.0
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENAI_API_MODEL: "gpt-4o" # Optional: defaults to "gpt-4"
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)